docs/querying/api.md

```diff
@@ -750,4 +769,45 @@ This takes no parameters or body.
 $ curl -XPOST http://localhost:9090/api/v1/admin/tsdb/clean_tombstones
 ```
 
+```json
+$ curl -XPUT http://localhost:9090/api/v1/admin/tsdb/clean_tombstones
+```
+
 *New in v2.1*
+
+## Prometheus Lifecycle APIs
```

Comment:
This page is scoped to /v1, which these aren't under. And apparently quit isn't documented anywhere. This seems like something to leave for another PR, as we'll need to find a space for these.

Reply:
Okay. I will remove this section